.github/workflows/pr-label-milestone-check.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Label and Milestone Check
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled, synchronize]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
```

Comment:
Should be using the self-hosted runner:

`runs-on: [self-hosted, Linux, medium, ephemeral]`